### PR TITLE
CPOL-102 Also provide policy create time.

### PR DESCRIPTION
**Pull request by pilhuhn**
- Fix a bug in mtime handling as well.

Signed-off-by: Heiko W. Rupp <hrupp@redhat.com>

### CODE REVIEW

**Comment by josejulio** [comment]
Looks good, but has conflicts now.

File: src/main/java/com/redhat/cloud/custompolicies/app/rest/PolicyCrudService.java [hunk 1]
```diff
@@ -434,11 +434,14 @@ public Response updatePolicy(@QueryParam ("dry") boolean dryRun, @PathParam("pol
 
           storedPolicy.populateFrom(policy);
           storedPolicy.customerid = user.getAccount();
+          storedPolicy.setMtimeToNow();
+          FullTrigger trigger = new FullTrigger(storedPolicy);
 
           existingTrigger.updateFromPolicy(storedPolicy);
```
Comment (josejulio) [inline src/main/java/com/redhat/cloud/custompolicies/app/rest/PolicyCrudService.java]: Is this not used anymore? What is it for? 
The `trigger` is not using the `updateFromPolicy`, so i assume we can remove that code?

Comment (pilhuhn) [reply src/main/java/com/redhat/cloud/custompolicies/app/rest/PolicyCrudService.java]: Yes, I think the `trigger` is rebase fallout. Will look at it later.

**Comment by pilhuhn** [comment]
@josejulio The `existingTrigger` is the one requested from the engine, which has the id fields set etc. We need to send this one back to the engine (in modified form).

**Pull request merged**